src/GeositeFramework/js/Export.js

```diff
@@ -54,63 +68,79 @@ require(['use!Geosite'],
                 maskopacity: 40,
                 openjs: function () {
```

Comment:
Both the `openjs` and `closejs` functions are long and complex enough that I think they warrant being broken out into their own named functions - and rescoping some of the above variables to just the functions where they're needed..  This would also reduce the level of indentation in the nested conditional.